src/foam/nanos/notification/email/EmailTemplate.js

```diff
@@ -148,7 +152,14 @@ foam.CLASS({
 
         // SUBJECT:
         if ( ! emailMessage.isPropertySet("subject") && ! SafetyUtil.isEmpty(getSubject()) ) {
```

Comment:
Are we assuming the caller is translating when explicitly setting the property?   yeah, I think that's ok.